glide-core/src/client/mod.rs

```diff
@@ -217,6 +218,42 @@ fn get_request_timeout(cmd: &Cmd, default_timeout: Duration) -> RedisResult<Opti
 }
 
 impl Client {
+    pub async fn cluster_scan<'a>(
```

Comment:
Add a doc

Reply:
As mentioned in the channel and on the PR itself - it is just a draft. It is not ready to review, it is just for getting the idea. Please wait with your review till i finish working on it, unless you see something critically wrong.